.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
             if [ "${CIRCLE_BRANCH}" == "develop" ]; then
               echo "$MAVEN_CENTRAL_SEC_RING" | base64 -d > $HOME/secring.gpg
               gpg --import --batch $HOME/secring.gpg
-              ./gradlew clean publish -Psonatype.username="$SONATYPE_USERNAME" -Psonatype.password="$SONATYPE_PASSWORD" -Psigning.keyId=0E7A8B89 -Psigning.password="$MAVEN_CENTRAL_KEY_PASSPHRASE" -Psigning.secretKeyRingFile=$HOME/secring.gpg -Porg.gradle.parallel=false -Porg.gradle.daemon=false
+              ./gradlew publish -PORG_GRADLE_PROJECT_mavenCentralUsername="$SONATYPE_USERNAME" -PORG_GRADLE_PROJECT_mavenCentralPassword="$SONATYPE_PASSWORD" -PORG_GRADLE_PROJECT_signingInMemoryKey=0E7A8B89 -PORG_GRADLE_PROJECT_signingInMemoryKeyPassword="$MAVEN_CENTRAL_KEY_PASSPHRASE" -Psigning.secretKeyRingFile=$HOME/secring.gpg -Porg.gradle.parallel=false -Porg.gradle.daemon=false
```

Comment:
`-Psigning.secretKeyRingFile=$HOME/secring.gpg`

I assume this needs to also change to `-PORG_GRADLE_PROJECT_signing.secretKeyRingFile` or somethin' like that to get picked up by the plugin